docs/change-log/index.md

```diff
@@ -528,6 +551,7 @@ with a drill-down to view the steps and their details. [Tech Preview]
 | ML-2014 | Model deployment returns ResourceNotFoundException (Nuclio error that Service <name> is invalid.) | Verify that all `metadata.labels` values are 63 characters or less. See the [Kubernetes limitation](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#syntax-and-character-set). |  v1.0.0  |
 | ML-3315 | The feature store does not support an aggregation of aggregations | NA                                  | v1.2.1 |
 | ML-3381 | Private repo is not supported as a marketplace hub             | NA                                     | v1.2.1 | 
+| ML-3824 | MLRun supports TensorFlow up to 2.9. | NA | v1.3.1 |
```

Comment:
~ 2.11.0 tbh

Reply:
@liranbg See https://iguazio.slack.com/archives/C056PBR36A2/p1684250135369859. I think the problem in the change log is that it's under limitataions, and not under 1.3.1 as a limitation. Is that correct?

Reply:
not sure I follow you. it is a limitation, could be 1.3.1 limitation only but tbh it will be 1.3.0 too now. because tensorflow was tested with < 2.12
anyway, this specific line should be changed to
```suggestion
| ML-3824 | MLRun supports TensorFlow up to 2.11. | NA | v1.3.1 |
```